config/locales/server.en.yml

```diff
@@ -2673,6 +2673,14 @@ en:
       user_api_key_allowed_groups: "min_trust_level_for_user_api_key"
       tag_topic_allowed_groups: "min_trust_level_to_tag_topics"
       profile_background_allowed_groups: "min_trust_level_to_allow_profile_background"
+      clean_up_inactive_users_after_days:
```

Comment:
This is a nice change, I am not sure why we didn't do this in the first place :thinking: